frontend/micro-ui/web/micro-ui-internals/packages/modules/orders/src/pages/employee/GenerateOrders.js

```diff
@@ -460,7 +460,7 @@ const GenerateOrders = () => {
 
   const isHearingAlreadyScheduled = useMemo(() => {
     const isPresent = hearingsData?.HearingList.some((hearing) => {
-      return !(hearing?.status === "COMPLETED" || hearing?.status === "ABATED");
+      return !(hearing?.status === "COMPLETED" || hearing?.status === "ABATED" || hearing?.status === "OPTOUT");
```

Comment:
this feels wrong,

if there is only one hearing and it is in optout status, then `isPresent` will be false, which would be wrong imo.

Let me know if you have an update